docs/contributor/03-60-regions-configuration.md

```diff
@@ -0,0 +1,44 @@
+# Regions and Zones Configuration
+
+Most Kyma plans require specifying a region in the request body as one of the provisioning parameters. The regions' configuration allows you to control which regions are available for the Kyma plan and the platform region. Additionally, every region has a different set of zones.
```

Comment:
Maybe in the last sentence, describe that below region configurations are used to create Kyma Environment Broker catalog schema for regions per plan.

That besides validtion on KEB side, KEB uses that for schema generation.